include/seqan3/argument_parser/detail/format_base.hpp

```diff
@@ -196,4 +195,257 @@ class format_base
     }
 };
 
+
+/*!\brief The format that contains all helper functions needed in all formats for
+ *        printing the interface description of the application (to std::cout).
+ * \ingroup argument_parser
+ */
+template <typename derived_type>
+class format_help_base : public format_base
+{
+private:
+    /*!\name Constructors, destructor and assignment
+     * \brief The (default) constructors.
+     * \{
+     */
+    format_help_base() = default;
```

Comment:
Don't you want to document the constructors? I have no idea what's happening here!